src/__tests__/render.test.js

```diff
@@ -267,38 +234,38 @@ test('getAllByDisplayValue, queryAllByDisplayValue', () => {
   expect(queryAllByDisplayValue('no value')).toHaveLength(0);
 });
 
-test('getByProps, queryByProps', () => {
-  const { getByProps, queryByProps } = render(<Banana />);
-  const primaryType = getByProps({ type: 'primary' });
+test('UNSAFE_getByProps, UNSAFE_queryByProps', () => {
```

Comment:
I think we should migrate our tests and examples to UNSAFE_ on 1.x, since people refer to our codebase for examples. WDYT?

Reply:
I don't know about this one. It is out internal thing. I think we can safely assume that if someone is looking for answers in the source code, they know what they're doing.